### ISSUE TITLE
False positive SlevomatCodingStandard.ControlStructures.ControlStructureSpacing.IncorrectLinesCountBeforeControlStructure

### ISSUE THREAD

**Issue by williamdes** [slevomat/coding-standard#1029]
```php
        if ($returnCode) {
            $status = curl_getinfo($curlHandle, CURLINFO_HTTP_CODE);// phpcs:ignore SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFallbackGlobalName

            return [
                $decodedResponse,
                $status,
            ];
        }
```
Bypass:
```php
        if ($returnCode) {
            $status = curl_getinfo($curlHandle, CURLINFO_HTTP_CODE);// phpcs:ignore SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFallbackGlobalName
            // A comment to bypass

            return [
                $decodedResponse,
                $status,
            ];
        }
```

**Comment by kukulich** [comment]
@williamdes I'm sorry I don't understand.

**Comment by williamdes** [comment]
Hi @kukulich 
Sorry I did not explain, `SlevomatCodingStandard.ControlStructures.ControlStructureSpacing.IncorrectLinesCountBeforeControlStructure` is reported for the return statement but the spacing is okay

If I add a comment the false positive goes away

**Comment by williamdes** [comment]
And the auto fixer is unable to fix the error, obviously

**Comment by kukulich** [comment]
@williamdes Do you use last version?

**Comment by williamdes** [comment]
This occurs here
https://github.com/phpmyadmin/error-reporting-server/blob/9561cd8ec40d712b94812d402c8cc9f277fe0a03/src/Controller/Component/GithubApiComponent.php#L144

[phpmyadmin/coding-standard composer.json](https://github.com/phpmyadmin/coding-standard/blob/c13eccca2dbaa1f4a382073da178b007209b34c6/composer.json#L20)

>>

[doctrine/coding-standard composer.json](https://github.com/doctrine/coding-standard/blob/903511eb53a7010f21c7ea423aaaeda041e54ffa/composer.json#L26)


They are not using the latest version.. (6.2.0)

A bot opened a PR: https://github.com/doctrine/coding-standard/pull/191

**Comment by kukulich** [comment]
@williamdes Are you able to test it with Slevomat CS 6.3.8?

**Comment by williamdes** [comment]
I could not reproduce this issue with the old version or a new one by copying their standard in another project.
Maybe you can close this one and hope it is fixed, it is surely the best option :)

Thank you for the quick support !
